content/applications/inventory_and_mrp/inventory/management/misc/owned_stock.rst

```diff
@@ -1,45 +1,106 @@
+From the Inventory main dashboard, click into the :guilabel:`Receipts` section. Then, click
+:guilabel:`New`.
+
+.. note::
+    Consignment stock is not actually purchased from the vendor; it is simply received and stored.
+    Because of this, there are no quotations or purchase orders involved in receiving consignment
+    stock. So, *every* receipt of consignment stock will start by creating manual receipts.
+
+Choose a vendor to enter in the :guilabel:`Receive From` field, then choose the same vendor to
+enter in the :guilabel:`Assign Owner` field.
+
+.. important::
+    Since the products received from the vendor will be owned by the same vendor, the
+    :guilabel:`Receive From` and :guilabel:`Assign Owner` field must match each other.
 
-.. image:: owned_stock/owned_stock_01.png
-    :align: center
+Once the vendor-related fields are set, enter products into the product lines, and set the
+quantities to be received into the warehouse under the :guilabel:`Done` column. Once everything has
+been set, :guilabel:`Validate` the receipt.
 
-Reception of Consignee Stock
-============================
+.. image:: owned_stock/owned-stock-receipt-fields.png
+   :align: center
+   :alt: Matching vendor fields in consignment Receipt creation.
 
-When in the *Inventory* app, open the receipts and create a new
-reception. On the right side, you will see that a new line called
-*Assign Owner* has appeared. There, you can specify the partner which
-owns the stock.
+Sell and Deliver Consignment Stock
+==================================
 
-.. image:: owned_stock/owned_stock_02.png
-    :align: center
+Once it's been received into the warheouse, consignment stock can be sold the same as any other
+product. To create a sales order, navigate to the :guilabel:`Sales` app, and from the
+**Quotations** overview, click :guilabel:`New`.
+
+Fill out all the information on the new quotation, and choose a customer to enter into the
+:guilabel:`Customer` field.
 
 .. note::
-   If you are the owner, you can leave the field blank.
+    The :guilabel:`Customer` *must* be different from the vendor that supplied the consignment
+    stock received into and stored in the warheouse.
+
+Add the consignment product to the product lines, set the :guilabel:`Quantity`, and once everything
+has been filled out, :guilabel:`Confirm`.
+
+.. image:: owned_stock/owned-stock-sales-order.png
+   :align: center
+   :alt: Sales order of consignment stock.
+
+Once the quotation has been confirmed, it will become a sales order. From here, the products can be
+delivered and invoiced for.
+
+Traceability and Reporting of Consignment Stock
+===============================================
+
+Although consignment stock is owned by the vendor who supplied it and not by the company storing
+it in their warehouse, consignment products will still appear in certain inventory reports. To find
+inventory reports, go to :menuselection:`Inventory --> Reporting`, and choose a report to view.
```

Comment:
Although consignment stock is owned by the vendor who supplied it, and not by the company storing
it in their warehouse, consignment products will *still* appear in certain inventory reports. 

To find inventory reports, go to :menuselection:`Inventory --> Reporting`, and choose a report to view.